### PR TITLE
Fixes and beauty

### PR DESCRIPTION
**Pull request by nikkrichko**
add some new functionality and add some beauty to console

### CODE REVIEW

File: cli/chat.go [hunk 1]
```diff
@@ -23,7 +24,8 @@ func AnimateThinking(quit chan bool) {
 		case <-quit:
 			return
 		default:
-			fmt.Printf("\rThinking %c", spinChars[i])
+			//fmt.Printf("\rThinking %c", spinChars[i])
```
Comment (b0noI) [inline cli/chat.go:27]: do not comment - remove

File: cli/chat.go [hunk 2]
```diff
@@ -86,7 +88,8 @@ func StartChat(rawContextId string, llmClient *client.Client, logger *logrus.Log
 	ShowMessages(messages)
 
 	for {
-		fmt.Print("You: ")
+		//fmt.Print("You: ")
```
Comment (b0noI) [inline cli/chat.go:91]: same

File: cli/chat.go [hunk 3]
```diff
@@ -114,7 +117,9 @@ func StartChat(rawContextId string, llmClient *client.Client, logger *logrus.Log
 			return err
 		}
 
-		fmt.Printf("\n\n\nJess: %s\n\n\n", response)
+		//fmt.Printf("\n\n\nJess: %s\n\n\n", response)
```
Comment (b0noI) [inline cli/chat.go:120]: same, do not comment - remove

File: commands_common/flags.go [hunk 4]
```diff
@@ -4,10 +4,10 @@ import "github.com/urfave/cli/v2"
 
 func PromptFlag() cli.Flag {
 	return &cli.StringFlag{
-		Name:     "prompt",
-		Aliases:  []string{"p"},
-		Usage:    "prompt to suppy with file",
-		Required: true,
+		Name:    "prompt",
+		Aliases: []string{"p"},
+		Usage:   "[optional] prompt to supply with file",
```
Comment (b0noI) [inline commands_common/flags.go:9]: I love it :)

File: commands_common/flags.go [hunk 5]
```diff
@@ -4,10 +4,10 @@ import "github.com/urfave/cli/v2"
 
 func PromptFlag() cli.Flag {
```
Comment (b0noI) [inline commands_common/flags.go:5]: for some of the cases prompt is not optional, so most likely you will need to pass bool as input here and than either make it required or not

File: commands_text/grammar_command.go [hunk 6]
```diff
@@ -0,0 +1,41 @@
+package commands_text
```
Comment (b0noI) [inline commands_text/grammar_command.go:1]: I love it :)

**Review by b0noI — APPROVED** [review]

**Comment by nikkrichko** [comment]
@b0noI  need to review your fixes. and merge it.

File: cli/chat.go [hunk 7]
```diff
@@ -86,7 +87,7 @@ func StartChat(rawContextId string, llmClient *client.Client, logger *logrus.Log
 	ShowMessages(messages)
 
 	for {
-		fmt.Print("You: ")
+		utils.Println_cyan(">>> You:")
```
Comment (b0noI) [inline cli/chat.go:90]: camel style

File: commands_common/jess_command.go [hunk 8]
```diff
@@ -50,7 +51,14 @@ func (c *JessCommand) handleAction(llmClient *client.Client) func(cliContext *cl
 				return err
 			}
 		} else {
-			fmt.Println("\n\n" + answer)
+			//fmt.Println("\n\n" + answer)
+			//TODO wrap changes with a function
```
Comment (b0noI) [inline commands_common/jess_command.go:55]: remove

**Review by b0noI — APPROVED** [review]

**Pull request merged**